tests/test_vcf.py

```diff
@@ -38,6 +38,8 @@ def test_validation_valid_no_samples(vcf_class):
             "QUAL": ["AAK1", "AAED1", "AAAS"],
             "FILTER": ["AAK1", "AAED1", "AAAS"],
             "INFO": ["AAK1", "AAED1", "AAAS"],
+            "FORMAT": ["AAK1", "AAED1", "AAAS"],
+            "TUMOR": ["AAK1", "AAED1", "AAAS"],
```

Comment:
This test is duplicate with the below one sample one. As the Jira ticket, for single sample vcf, we check the existence of either Tumor or Sample column. I would update the one sample test with sample column.

Reply:
Actually since this is just for testing the entire _validate function, I'd just remove this because we already have tests for all of the individual scenarios for the specific tumor/normal validation function